flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaProducerMigrationTest.java

```diff
@@ -44,7 +44,7 @@
 	@Parameterized.Parameters(name = "Migration Savepoint: {0}")
 	public static Collection<MigrationVersion> parameters() {
 		return Arrays.asList(
-			MigrationVersion.v1_8);
+			MigrationVersion.v1_9);
```

Comment:
I think we should also keep `MigrationVersion.v1_8` here.

Reply:
I am not against it. But in many test cases, we updated the migration version. Why not update this?

Reply:
I think we always added the new version `v1_9` to the set of tested versions if there wasn't an incompatibility. Here we deactivate the test which tests the migration from checkpoints of version `1.8`

Reply:
OK, I will reset it to `v1_8`.

Reply:
Done

Reply:
I think it should be:
```suggestion
                         MigrationVersion.v1_8,
			MigrationVersion.v1_9);
```